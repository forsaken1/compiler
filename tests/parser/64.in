int f(int a = 0, float b, char c) {
    while(a > 0)
	b += a;
}