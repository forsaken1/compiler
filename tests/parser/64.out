int function (f)
|
+---(,)
|   |
|   +---(=)
    |   |
    |   +---(a)
    |   |
    |   +---(0)
|   |
|   +---(,)
        |
        +---(b)
        |
        +---(c)
|
+---{stmt}
    |
    +---while
        |
        +---(>)
        |   |
        |   +---(a)
        |   |
        |   +---(0)
        |
        +---(+=)
            |
            +---(b)
            |
            +---(a)